### PR TITLE
added rajdhani font to the app.css and index.html, also updated the f…

### LINKED ISSUE #40: minor tweak to followup
If a user has no job follow ups for a specific company, the app should display no followups 
Importing the wyncode font from google fonts as well.

### PR DESCRIPTION
**Pull request by Rterrell25**
…ollowups to display No job followups for a specific company if there are none.

closes #40

### CODE REVIEW

**Comment by Rterrell25** [comment]
Screen Shot 2020-03-09 at 2 04 02 PM

File: src/App.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ import { ProfileContext } from './contexts/ProfileContext'
 
 const theme = createMuiTheme(themeFile)
 
-axios.defaults.baseURL = `https://us-central1-jobtracker-4f14f.cloudfunctions.net/api`
+// axios.defaults.baseURL = `https://us-central1-jobtracker-4f14f.cloudfunctions.net/api`
```
Comment (JamesKemp296) [inline src/App.js]: Cannot merge with this being commented out. Will break deployment.

File: src/components/JobCard.js [hunk 2]
```diff
@@ -319,7 +321,10 @@ const JobCard = ({
                   </Grid>
                 </div>
               </Card>
-            ))}
+            ))
+          ) : (
+            <p>No Job Followups with {company}</p>
```
Comment (JamesKemp296) [inline src/components/JobCard.js]: Looks nice but lets try to use `<Typography/>` only. Add fonts to the theme in App.js

File: src/App.css [hunk 3]
```diff
@@ -1,9 +1,7 @@
 html,
 body {
   background-color: rgb(245, 245, 245);
```
Comment (JamesKemp296) [inline src/App.css]: Let's not set the font on the entire body, as it should be added per element.

**Pull request merged**